toolchain/sem_ir/file.h

```diff
@@ -304,8 +298,23 @@ class File : public Printable<File> {
   auto set_has_errors(bool has_errors) -> void { has_errors_ = has_errors; }
 
  private:
+  // Allocates a copy of the given data using our slab allocator.
+  template <typename T>
+  llvm::MutableArrayRef<T> AllocateCopy(llvm::ArrayRef<T> data) {
```

Comment:
Please switch to a trailing return style.

Reply:
Done.